Gemfile

```diff
@@ -11,6 +11,8 @@ gem "decidim-initiatives", path: "."
 
 gem "bootsnap", "~> 1.3"
 
+gem "doc2text", git: "https://github.com/tramuntanal/doc2text.git", branch: "feature/markdown_headers"
```

Comment:
The old gem doesn't seem to be maintained, if using this fork is the only way I'd say it would be better to move the repo to de Decidim organization so you don't have to maintain it/worry about it on the future.

Reply:
I'm in talks with the creator and have already a fork with fixes and completions. If he doesn't give feedback soon I'll definitively create the new repo. Indeed we also would need to create a new derivative gem (or you are thinking to directly add the code to decidim main project?).

Reply:
>  Indeed we also would need to create a new derivative gem (or you are thinking to directly add the code to decidim main project?).

I'd prefer creating a new repo just for it!

Reply:
Finally I've been given collaborator permissions for doc2text so we'll be able to keep on working with the original gem.
Can you accept this PR while we're working in supporting docx and release a new doc2text gem version?

Reply:
@oriolgual Any feedback here?

Reply:
I didn't see the change to the original repo, works for me!